src/compositions/templates/case-study/Success/spec.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Success from './index';
+
+test('success section', () => {
+  const wrapper = shallow(
+    <Success
+      success={{
```

Comment:
Could we have two props, one being `main` and one being `sub` instead of one prop that is an object?